lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -320,28 +336,30 @@ class CategoryRenderer {
   /**
    * @param {!ReportRenderer.CategoryJSON} category
    * @param {!Object<string, !ReportRenderer.GroupJSON>} groups
+   * @param {!Object<string, !ReportRenderer.CertificationJSON>} certifications
    * @return {!Element}
    */
-  render(category, groups) {
+  render(category, groups, certifications) {
```

Comment:
I've been thinking that we should be passing the `report` object around instead of adding additional params to all of the render methods + piping it throughout. Each method can pull the properties it cares about. WDYT?